wa/workloads/speedometer/__init__.py

```diff
@@ -24,15 +24,15 @@
 import time
 import uuid
 
-from wa import Parameter, Workload, File
+from wa import ApkUiautoWorkload, Parameter, Workload, File
 from wa.framework.exception import WorkloadError
 from wa.utils.exec_control import once
 from wa.utils.misc import safe_extract
 
 from devlib.utils.android import adb_command
 
 
-class Speedometer(Workload):
+class Speedometer(ApkUiautoWorkload):
```

Comment:
Just to add some background to this workload you may not be aware of.
 
A previous iteration of this workload used to be an `ApkUiautoWorkload` however after an investigation it seems that using UIAutomator introduced a performance overhead when running this workload as compared the current version [1].

It would be worth trying to confirm if this is still an issue.

[1] https://github.com/ARM-software/workload-automation/commit/7489b487e1fa808a793581617b0f8ed2b303f491#diff-83da7fd8a8e5e6b156a68f2d37658c2468594da1ceb7f48d0720a6f5324540ee

Reply:
Hi,
The UiAutomator is only for chrome itself and Speedometer still runs offline. To dismiss the pop-ups coming from Chrome, I have added UiAutomator to `setup` and `extractResults` stages only. When the workload is in the `runWorkload` stage, there is no UiAutomator. That's why I believe I didn't see any drop on the score. 
Or I can try to find out another solution without UiAutomator.